server/chat-plugins/trivia.ts

```diff
@@ -1726,7 +1763,9 @@ const commands: ChatCommands = {
 			`- <code>/ta [answer]</code> - Answer the current question.`,
 			Utils.html`- <code>/trivia kick [username]</code> - Disqualify a participant from the current trivia game. Requires: % @ # &`,
 			`- <code>/trivia leave</code> - Makes the player leave the game.`,
-			Utils.html`- <code>/trivia end</code> - End a trivia game. Requires: + % @ #`,
+			Utils.html`- <code>/trivia end</code> - End a trivia game. Requires: + % @ # &`,
+			Utils.html`- <code>/trivia pause</code> - Pauses a trivia game. Requires: + % @ # &`,
+			Utils.html`- <code>/trivia resume</code> - Resumes a paused trivia game. Requires: + % @ # &`,
 			``,
 			`<strong>Question-modifying commands:</strong>`,
 			Utils.html`- <code>/trivia submit [category] | [question] | [answer1], [answer2] ... [answern]</code> - Adds question(s) to the submission database for staff to review. Requires: + % @ # &`,
```

Comment:
Hm, none of these actually require `Utils.html`.

```suggestion
			`- <code>/trivia end</code> - End a trivia game. Requires: + % @ # &`,
			`- <code>/trivia pause</code> - Pauses a trivia game. Requires: + % @ # &`,
			`- <code>/trivia resume</code> - Resumes a paused trivia game. Requires: + % @ # &`,
			``,
			`<strong>Question-modifying commands:</strong>`,
			`- <code>/trivia submit [category] | [question] | [answer1], [answer2] ... [answern]</code> - Adds question(s) to the submission database for staff to review. Requires: + % @ # &`,
```

We should probably standardize on just not having it?